.bundle/config

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
It's not surprising: it's just because it used to be `.gitignore`'d (but now we want to commit it to have the same config for everyone).

I think the right move is to use the one from the PR replace the one you had locally. (This might make people have `git pull` error about the conflicting files once the PR is merged tho, might need to warn them in Slack)